src/scripts/components/Header.jsx

```diff
@@ -43,6 +43,10 @@ export default class Header extends React.Component {
     });
   }
 
+  goHome() {
+    window.location.replace("/");
```

Comment:
Is it expected that the user won't be able to navigate back to the previous page?  I think that's what happens when you use `replace` rather than `assign`, but I'm not very experienced with that.

Reply:
That's a good point. I'll change it to assign and test that way. I'm guessing people would like to navigate back.